pkg/commands/util.go

```diff
@@ -32,12 +32,15 @@ func GetThreadUrl(event *slackevents.MessageEvent) string {
 			workspace, event.Channel, strings.Replace(event.ThreadTimeStamp, ".", "", 1))
 
 		return threadURL
-	} 
+	}
 	log.Println("This is not a threaded message")
-	return ""	
+	return ""
 }
 
 func ContainsBotMention(messageText string) bool {
-	return strings.Contains(messageText, fmt.Sprintf("<@%s>", BOT_USER_ID))
+	userID, ok := os.LookupEnv("SPLAT_BOT_USER_ID")
+	if !ok {
+		log.Fatal("no bot user id specified with SPLAT_BOT_USER_ID")
```

Comment:
should we return an error from here?

Reply:
`log.Fatal` will `exit(1)`

I can change it to bubble up the error instead, but should we exit or continue?

Reply:
id vote to continue, im not sure we want the bot to exit